### PR TITLE
feat(lpa-list): lpa list working as an accessible autocomplete

### PR DESCRIPTION
**Pull request by mmuchlinski**
Local Planning Department / Authority : Autocomplete input

## Changes

- added js. for Accessible Autocomplete: forms-web-app/src/public/javascript/accessible-autocomplete.min.js
- added css for Accessible Autocomplete: forms-web-app/src/public/javascript/stylesheets/accessible-autocomplete.min.css
- added planning department view: forms-web-app/src/views/eligibility/planning-department.njk
-removed forms-web-app/src/controllers/local-planning-authorities.js functionality handled in forms-web-app-src-controllers-eligibility-planning-department.js
- added routes planning-department: forms-web-app/src/routes/eligibility/planning-department.js
- added: forms-web-app/.prettierignore
-test for planning-department updated to include data passed in

## PR Template Checklist

- [ ] Grammar
- [ ] Spelling
- [ ] Clarity

Fixes #85, Fixes #22, Fixes username/repo#123
Connects #123

### CODE REVIEW

File: forms-web-app/src/views/eligibility/planning-department.njk [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
 
-{% endblock %}
+  <script>
+    accessibleAutocomplete({
+      element: document.querySelector('#planning-department'), id: 'planning-department-label', // To match it to the existing <label>.
+      source: {{ data|safe }}
+    });
+  </script>
+
+{% endblock %}
```
Comment (MrSimonEmms) [inline forms-web-app/src/views/eligibility/planning-department.njk:31]: Add newline at end of file

File: forms-web-app/.eslintignore [hunk 2]
```diff
@@ -1 +1,2 @@
 node_modules
+arc/public
```
Comment (MrSimonEmms) [inline forms-web-app/.eslintignore:2]: Add newline at end of file

File: forms-web-app/src/controllers/eligibility/planning-department.js [hunk 3]
```diff
@@ -1,3 +1,16 @@
+const PLANNING_DEPARTMENT = 'eligibility/planning-department';
+
+const LPAS = [
```
Comment (MrSimonEmms) [inline forms-web-app/src/controllers/eligibility/planning-department.js:3]: Happy for this to go in like this, but please create a ticket to get data from the appeals service API

File: forms-web-app/src/views/eligibility/planning-department.njk [hunk 4]
```diff
@@ -1,30 +1,31 @@
...
-          name: "local-planning-department"
-        }) }}
-
-        {% from "govuk/components/button/macro.njk" import govukButton %}
-
-        {{ govukButton({
-          text: "Continue",
-          type: "Submit"
-        }) }}
+  <div class="govuk-grid-row">
```
Comment (MrSimonEmms) [inline forms-web-app/src/views/eligibility/planning-department.njk:6]: LGTM but I'll defer to @a6software on this section

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]

File: forms-web-app/.eslintignore (hunk 2 above)
Comment (MrSimonEmms) [reply forms-web-app/.eslintignore:2]: `src` not `arc`

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]

File: forms-web-app/.eslintignore (hunk 2 above)
Comment (Fenisay) [reply forms-web-app/.eslintignore:2]: If you don't have it, I advise you to use the plugin "prettier" which is automatically doing formatting/indentations following the good practices rules. Made my life more easier and prettier.

I don't which IDE you are using but it's available on most of them.

Comment (MrSimonEmms) [reply forms-web-app/.eslintignore:2]: Yeah, not sure why it's not working on @mmuchlinski's machine. There's an `.editorconfig` file in the root which should be working out the box on Atom as well

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
Merge #93 in (and potentially squash) before going to `master`

**Review by a6software — APPROVED** [review]
Merge as is, I am addressing the jquery / autocomplete package issue

File: forms-web-app/src/views/includes/scripts.njk [hunk 5]
```diff
@@ -1,2 +1,2 @@
 <!-- Javascript -->
-<script src="/public/javascripts/jquery-1.11.3.js"></script>
+<script src="/public/javascript/jquery-1.11.3.js"></script>
```
[the remark this answers is not in the document]
Comment (mmuchlinski) [reply forms-web-app/src/views/includes/scripts.njk:2]: Not needed for the autocomplete, probably safe to remove, but worth checking.

**Pull request merged**